lib/src/kvs/sled/bench.rs

```diff
@@ -0,0 +1,193 @@
+#[cfg(test)]
+mod tests {
+	use crate::kvs::{Key, Val};
+	use rand::Rng;
+	use std::collections::{BTreeMap, HashSet};
+	use std::time::SystemTime;
+
+	const N: usize = 1000;
+	const M: usize = 1000;
+	const KEY: Key = Key::new();
+	const VAL: Val = Val::new();
+
+	fn get_randomized_map(random: u16) -> BTreeMap<Key, Val> {
+		let mut map = BTreeMap::<Key, Val>::new();
```

Comment:
Type hints are not necessary here. `BTreeMap::new()` should be enough.

Reply:
Definitely!